docs/WIP/Tutorial_Motif_only_Matrix_Profile.ipynb

```diff
@@ -0,0 +1,839 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #29.        mp, _ = stumpy.mstump(stacked_segments, m)  # Compute the matrix profile for the stacked segments
</pre><p>mstump is multi-dimensional stump. Does the paper try to analyze the two segments in multi-dimensional way? (I do not know what that means here since <code>mstump</code> is complicated) </p><p> </p><p>Or, does it try to compute the mp for the subsequences in those two segments, something like <code>stumpy.stump(np.r_[segA, np.nan, segB], m)</code> ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/1046/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>Thanks @nimasarajpoor. I totally missed this. I'm guessing that <code>msump</code> is NOT what we want here! </p>